Sources/TripKitUI/managers/TKUITripMonitorManager.swift

```diff
@@ -166,9 +166,14 @@ extension TKUITripMonitorManager {
     
     guard !pairs.isEmpty else { return }
     
-    geoMonitor.enableInBackground = true
```

Comment:
Didn't notice `enableInBackground` was set already when `enabledKey` is initialized